.travis.yml

```diff
@@ -1,14 +1,10 @@
 sudo: required
 
-language: csharp
-
-mono:
-  - latest
+rvm:
+  - 2.2.1
 
 
```

Comment:
What do you think about removing those extra spaces? Kinda picky but I think that one single space will be enough to provide a proper separation between steps. _Travis documentation uses a single space to separate steps._